### PR TITLE
feat-learn-css-grid

### PR DESCRIPTION
**Pull request by padunk**
Initial code for learn-css-grid-by-building-a-magazine.

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
Thanks for your patience @padunk. I'll take a look at these steps and will leave some review comments before merging.

File: learn-css-grid-by-building-a-magazine/css/style240.css [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.social-icons {
+  font-size: 3rem;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-auto-flow: column;
+  grid-auto-columns: 1fr;
+  align-items: center;
+}
+
+/* To divide you magazine to columns, you can use `column-width` property. This is not a grid property, but a common css property and very useful to divide your column into smaller columns.
```
Comment (scissorsneedfoodtoo) [inline learn-css-grid-by-building-a-magazine/css/style240.css]:
```
To divide you magazine to columns, use the `column-width` property. This is not a CSS Grid property, but a common CSS property that's very useful for dividing columns into smaller columns.
In the `.text` rule, set `column-width` to `25rem`.
```

File: learn-css-grid-by-building-a-magazine/css/style250.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+.social-icons {
+  font-size: 3rem;
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  grid-auto-flow: column;
+  grid-auto-columns: 1fr;
+  align-items: center;
+}
+
+/* Looking much better, but how to tidy it up a little bit? Use `text-align` property and set it to `justify`.
```
Comment (scissorsneedfoodtoo) [inline learn-css-grid-by-building-a-magazine/css/style250.css]:
```
Things are looking much better, but we can improve the layout of the text.
Use the `text-align` property and set it to `justify`.
```

**Comment by padunk** [comment]
I am already done revised the code.
Sorry for the late response. I will add some more steps before PR everything.
Thank you.

**Comment by scissorsneedfoodtoo** [comment]
@padunk, no need to apologize. Thank you for all of your hard work on these revisions and steps. Feel free to submit another PR whenever you're ready.

**Pull request merged**